frontend/src/Activity/Queue/QueueRow.js

```diff
@@ -100,6 +100,7 @@ class QueueRow extends Component {
       outputPath,
       downloadClient,
       estimatedCompletionTime,
+      addedOn,
```

Comment:
Should this be named `addedOn` or `added`?

Reply:
I think `added`/`Added` is more consistent with existing fields.